### ISSUE TITLE
Source of CLIP emotion classifier

### ISSUE THREAD

**Issue by guoyanan1g** [JingyuanYY/EmoGen#7]
hello, thx for your work! I noticed that the emo classifier (which turn clip-embedding dim 768 to dim 8) is trained and the weight is ready, but I don't find the training code of it.How did you get it?Should it be trained on specfic dataset?

**Comment by fengjw0909** [comment]
The training of this emotion classifier is quite simple. It involves extracting image features using CLIP, passing them through a fully connected (FC) layer for classification, with the training dataset being EmoSet.

**Comment by guoyanan1g** [comment]
I get it,thx!
